### PR TITLE
Nb config view page

### LINKED ISSUE #1313: Implement NodeBalancer config view page
Nodes list here:
https://projects.invisionapp.com/share/8FALUJ0EY#/screens/220575693

### PR DESCRIPTION
**Pull request by abemassry**
closes #1313

### CODE REVIEW

**Comment by eatonphil** [comment]
Could you clean up my mess for me by s/configurations/configs everywhere in NodeBalancers please? Sorry about that.

File: src/nodebalancers/nodebalancer/configs/layouts/ViewConfigPage.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          >
+            <List>
+              <ListBody>
+                <Table
+                  className="Table--secondary"
+                  columns={[
+                    { textKey: 'label', label: 'Label',
+                      cellComponent: LinkCell,
+                      hrefFn: function (node) {
+                        const string = [];
```
Comment (mparke) [inline src/nodebalancers/nodebalancer/configs/layouts/ViewConfigPage.js]: fwiw, this is more succint w/:
```
return [
  '/nodebalancers',
  ...
].join('/');
```

Comment (mparke) [inline src/nodebalancers/nodebalancer/configs/layouts/ViewConfigPage.js]: @na3d made a good point re: having to create the array every time, 

could also make one line and disable linting?

**Comment by mparke** [comment]
Follow ups:

https://github.com/linode/manager/issues/1666
https://github.com/linode/manager/issues/1667

**Pull request merged**